Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -188,12 +191,15 @@ private void RemoveRenderer(int index)
         /// </summary>
         public void ClearRenderers()
         {
-            if (renderers != null)
+            if (renderers.Count > 0)
```

Comment:
I have a faint memory of renderers being null is some editor scenarios. Unity may have addressed this since? But, work testing.

Reply:
Renderers collection is now readonly. In my testing I have never seen it being null.